Migration.Toolkit.CLI/README.md

```diff
@@ -138,6 +139,26 @@ By default, JSON data storing the Page Builder content of pages and custom page
 
 The Migration toolkit provides an advanced migration mode for Page Builder content that utilizes API discovery on the source instance. To learn more details and how to configure this feature, see [Source instance API discovery](#source-instance-api-discovery).
 
+#### Categories
+
+Xperience by Kentico uses a different approach to categorization than older Kentico versions. [Categories](https://docs.kentico.com/13/configuring-xperience/configuring-the-environment-for-content-editors/configuring-categories) were replaced by [taxonomies](https://docs.kentico.com/developers-and-admins/configuration/taxonomies) and selected categories for each page are assigned to pages in the target instance via a [reusable field schema](https://docs.kentico.com/x/D4_OD). The key differences are:
+
+* Categories in older versions can be added to any page via the *Properties -> Categories* tab. Taxonomies can only be added to content items (pages, emails...) that have a field with the *Taxonomy* data type configured to select tags from a certain taxonomy.
+* Categories can be global or site-specific. Taxonomies are always global, as there are no sites in Xperience by Kentico.
+* Categories are assigned to pages regardless of their workflow step. Taxonomies are stored as a field and are covered by the workflow, therefore selected tags can be different in each workflow step.
```

Comment:
```suggestion
* Categories are assigned to pages regardless of their workflow step. Taxonomies are stored as a field and are covered by workflow. As a result, assigned tags can be different in each workflow step.
```

add link to workflows - https://docs.kentico.com/developers-and-admins/configuration/workflows